test/css-skins/ui-test.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>UI Test</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script type="text/javascript" src="../../bin-debug/jwplayer.js"></script>
+</head>
+<body style="width: 100%; height: 2000px">
+    <div class="jwplayer playlist-none jw-user-inactive" tabindex="0" style="width: 854px; height: 480px; opacity: 1; background-color: rgb(0, 0, 0);">
```

Comment:
Can "playlist-none" be removed?

Which of these inline styles can also be removed?